README.md

```diff
@@ -39,6 +39,13 @@ by the [AppLand CLI](https://github.com/applandinc/appland-cli/releases).
 Then, on the [AppLand website](https://app.land), you can
 visualize the design of your code and share links with collaborators.
 
+### Supported versions
+
+* Ruby 2.5, 2.6
+* Rails 4, 5, 6
```

Comment:
Kevin's got a PR open that removes Rails 4, because it's been EOLed. This should just be `Rails 5, 6` now.